### ISSUE TITLE
Switch to jekyll + Feeling responsive

### ISSUE THREAD

**Issue by DanySK** [Protelis/Protelis.github.io#7]
We may want to change the way we are organizing our information on the website, along with its look and feel.
I learned how to configure and use Jekyll, and I believe that along with [Feeling responsive](https://github.com/Phlow/feeling-responsive) it can produce fully functional and nice static websites. I have used these tools for [the Alchemist website](http://danysk.github.io/alchemist/).

Pluses of this choice:
* We get a navigation menu, easy to configure through a yml file
* Pages are written in markdown, and are very easy to maintain wrt the current HTML setup
* Support for Disqus if we want users to comment (e.g. for telling us that some part could be improved)

Minuses of this choice:
* The current content should be copy/pasted there
* There is some theming work to do

@jakebeal please let me know if there are any other points you see

**Comment by jakebeal** [comment]
I've just been starting to figure out Jekyll myself, to see if making such a switch would be advisable.  I agree that we should make the switch, assuming we can figure out how to do the theming.

**Comment by DanySK** [comment]
If you like the time style of Alchemist, I can use the same one (with different logos and imagery) for Protelis. Also, it has the advantage of showing that the two projects are somehow related.

**Comment by jakebeal** [comment]
Yes, I like it.  Let's do it.  Let me know when it's in, and I'll start contributing too.
